OpenChange/MAPIStoreMailFolder.m

```diff
@@ -257,6 +257,16 @@ - (int) getPidTagContainerClass: (void **) data
   return MAPISTORE_SUCCESS;
 }
 
+
+- (int) getPidTagDisplayName: (void **) data
+                    inMemCtx: (TALLOC_CTX *) memCtx
+{
+  NSString *displayName = [[sogoObject displayName] stringByDecodingImap4FolderName];
```

Comment:
I think we should first `fromCSSIdentifier` and then `stringByDecodingImap4FolderName`